fedimint-client/src/lib.rs

```diff
@@ -61,6 +61,12 @@ pub mod sm;
 /// Structs and interfaces to construct Fedimint transactions
 pub mod transaction;
 
+/// Client backup
+pub mod backup;
+
+// TODO: where should this sit?
+pub const BACKUP_CHILD_ID: ChildId = ChildId(1);
```

Comment:
@elsirion What do you think?

Reply:
@elsirion Now we have one backup for all modules.

We should probably have:

* root / TYPE_MODULE / module_id
* root / TYPE_BACKUP 

kind of thing?